packages/amplify-migration-codegen-e2e/src/index.ts

```diff
@@ -40,6 +46,27 @@ export async function setupAndPushGen1Project(projRoot: string, projName: string
   await amplifyPushForce(projRoot);
 }
 
+export async function setupAndPushAuthWithMaxOptionsGen1Project(projRoot: string, projName: string) {
```

Comment:
I don't see you add external providers (Amazon, Facebook...) but see your setup of secrets?

Reply:
I included that as part of running codegen command function which had npm install which is required before running. I will separate all those commands and use it wherever necessary.

Reply:
Done [dbf44b4](https://github.com/aws-amplify/amplify-cli/pull/13978/commits/dbf44b417f56692be3608d684dc8f31f985b6df8)